doc/programming_guide/media.rst

```diff
@@ -95,25 +119,102 @@ DirectSound is available only on Windows, and is installed by default.
+By default, this is the most preferred driver. It will be used first
+if it is installed unless overridden. See :ref:`guide-audio-driver-order`
+to learn more.
 
-Pulse
-^^^^^
+It is the best target for cross-platform consistency due to being either
+preinstalled or easy to install on all supported platforms. It implements
+software versions of common game audio features such as positional mixing
+on platforms which may not support them by default.
+
+OpenAL's main downside is that platforms other than Mac OS X are not
+guaranteed to have it preinstalled.
+
+Windows users can download a generic driver from either `openal.org`_
+or their sound device's manufacturer.
+
+On Linux, it may already be installed. It is strongly recommended to
+install it since the :ref:`guide-pulseaudio` backend has serious bugs
+and limitations. The commands to install OpenAL on the most common
+Linux distros are listed below. You may need to use ``sudo`` to run
+them.
+
+.. list-table::
+    :header-rows: 1
+
+    * - Common Linux Distros
+      - Install Command
+
+    * - Ubuntu, Pop!_OS, Debian
+      - ``apt install libopenal1``
+
+    * - Arch, Manjaro
+      - ``pacman -S openal``
+
+    * - Fedora, Nobara
+      - ``dnf install openal-soft``
+
+These commands should be safe to run when OpenAL is already
+installed. You will be informed the package is already installed
+if this is the case, and no action will take place.
+
+Note that OpenAL does not appear to have a package on recent versions of
+RHEL.
+
+.. _guide-pulseaudio:
+
+PulseAudio
+^^^^^^^^^^
+
+.. warning:: Install and use :ref:`guide-openal` instead of this driver
+             whenever possible!
+
+Although most modern Linux distributions support PulseAudio, this pyglet
+driver has serious downsides:
+
+#. It contains a bug which can crash your program
+#. PulseAudio lacks built-in support for standard game audio features
+
+The bug
+"""""""
+
+The bug is known to crash programs when one or more of the following
+occurs:
+
+#. A debugger pauses or resumes the program
+#. Unpredictably when 2 or more sounds are playing
+
+It is not known to be affected by whether the PulseAudio implementation
+is the stand-alone or PipeWire compatibility layer.
+
+Do not expect the bug to be fixed quickly for multiple reasons:
+
+#. :ref:`guide-openal` is better in almost every way
+#. It unpleasant to fix because it appears to be a concurrency problem.
+   See the `original github issue <https://github.com/pyglet/pyglet/issues/952>`_
+   and discussion below it for more information.
+#. PulseAudio is on its way to being replaced by PipeWire, so replacing
+   this driver may be a better use of developer time than fixing it.
+
+Missing features
+""""""""""""""""
+
+PulseAudio was designed with only desktop computer workloads in mind.
+This means it lacks built-in support for common features such as spatial
+audio mixing. The following features will not work properly with this
+driver:
 
```

Comment:
This might read as being misleading to some people, since PulseAudio itself does have support for advanced multi-channel audio.

It would be more accurate to say that it's pyglet's PulseAudio backend that is limited to stereo.